runner/app/routes/utils.py

```diff
@@ -77,6 +77,16 @@ class LLMMessage(BaseModel):
     content: str
 
 
+class LLMChoice(BaseModel):
+    delta: LLMMessage
+    index: int
+    finish_reason: str = ""
```

Comment:
Is this default by design? AFAIK It will make all the `LLMChoice` objects in responses to contain an empty `finish_reason` field, instead of omitting the field when it's not a final chunk or sth

Reply:
Here's the generated go binding for the type, which would omit it when empty. It seems counterintuitive but it works.

```
type LLMChoice struct {
	Delta        LLMMessage `json:"delta"`
	FinishReason *string    `json:"finish_reason,omitempty"`
	Index        int        `json:"index"`
}
```

Reply:
It does return `finish_reason: ""` in the response however